### DISCUSSION TITLE
Sort by reactions

### DISCUSSION THREAD

**Discussion by janpio** [community/community#10463] [Discussions]
Discussions have both Upvotes and Reactions to the main message. 
It would be great if one could sort by reactions similar you can sort for them in Issues.

**Comment by wdawson** [comment]
This is especially needed when migrating from Issues to Discussions. I don't have a way to understand the top requests/ideas anymore.

**Comment by ghost** [comment]
Hi all! I have an idea that is similar to what you are asking for! please, see here: https://github.com/github/feedback/discussions/16617 - My idea is to order the discussions by the amount of comments - would that be similar to sort by reactions here?
